src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -132,6 +132,12 @@ ReportAie::writeReport(const xrt_core::device* /*_pDevice*/,
 
 	if(tile.second.find("dma") != tile.second.not_found()) {
           _output << boost::format("    %s:\n") % "DMA";
+          _output << boost::format("        %s:\n") % "FIFO";
+          for(auto& node : tile.second.get_child("dma.fifo_info")) {
+            _output << fmt16("%s") % "Size" % node.second.get<std::string>("fifo_size");
+            _output << fmt16("%s") % "Count" % node.second.get<std::string>("fifo_count");

```

Comment:
By looking at following registers, It looks like these are fifo counter0 and fifo counter1 values.
By looking at following spec
http://cervino-doc/r2p19/tile_links/xregdb_me_tile_doc.html#tile_memory___DMA_FIFO_Counter
i

Reply:
You can safely assume that, there are two fifo counters.

Reply:
Fixed. Thanks